media_kit/lib/src/player/web/utils/hls.dart

```diff
@@ -0,0 +1,92 @@
+/// This file is a part of media_kit (https://github.com/media-kit/media-kit).
+///
+/// Copyright © 2021 & onwards, WanJiMi.
```

Comment:
I added your name to this source file, in-case any change is required, raise a PR in future.